src/features/movieDetails/index.js

```diff
@@ -1,57 +1,143 @@
+              Accept: "application/json",
+            },
+          }
+        );
+
+        if (!responseMovie.ok) {
+          throw new Error(responseMovie.statusText);
+        }
+
+        const result = await responseMovie.json();
+        dispatch(setLoadingState("success"));
+        setMovieData(result);
+      } catch (error) {
+        dispatch(setLoadingState("error"));
+        console.error("Error fetching movie details:", error);
+      }
+    };
+    fetchMovie();
+  }, [id, dispatch]);
+
+  return loadingState === "loading" ? (
+    <LoadingSpinner />
+  ) : (
+    movieData && (
+      <>
+        <TopTileContainer
+          imageSrc={
+            movieData.backdrop_path
+              ? `https://image.tmdb.org/t/p/w500${movieData.backdrop_path}`
+              : null
+          }
+          title={movieData.original_title}
+          rate={movieData.vote_average || 0}
+          vote={movieData.vote_count || 0}
+        />
+        <MovieDetailsWrapper
+          imageSrc={
+            movieData.poster_path
+              ? `https://image.tmdb.org/t/p/w500${movieData.poster_path}`
+              : null
+          }
+          title={movieData.original_title}
+          yearShort={movieData.release_date.substring(0, 4)}
+          production={movieData.production_countries.map(country => country.name).join(', ')}
+          yearFull={movieData.release_date}
+          categories={movieData.genres.map(genre => genre.name)}
+          rate={movieData.vote_average || 0}
+          vote={movieData.vote_count || 0}
+          description={movieData.overview}
+        />
+        <CastCrewWrapper>
+          <Header>Cast</Header>
+          <TilesContainer persontile="true">
+            {movieData.credits.cast &&
+              movieData.credits.cast.map((castMember) => {
+                return (
+                  <NavLink
+                    key={castMember.id}
+                    to={toProfile({ id: castMember.id })}
+                    style={{ textDecoration: "none" }}
+
+                  >
+                    <PersonTile
+                      imageSrc={
+                        castMember.profile_path
+                          ? `https://image.tmdb.org/t/p/w500${castMember.profile_path}`
+                          : null
+                      }
+                      key={`${castMember.name}`}
+                      character={castMember.character}
+                      name={castMember.name}
+                    />
+                  </NavLink>
+                );
+              })}
+          </TilesContainer>
+        </CastCrewWrapper>
+        <CastCrewWrapper second={`${true}`}>
+          <Header>Crew</Header>
+          <TilesContainer persontile="true">
+            {movieData.credits.crew &&
+              movieData.credits.crew.map((crewMember) => {
+                return (
+                  <NavLink
+                    key={crewMember.id}
+                    to={toProfile({ id: crewMember.id })}
```

Comment:
Do we send user to details of person from this page. Did Chris say anything about it?